src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -178,10 +179,31 @@
     /** Period literal. */
     private static final String PERIOD = ".";
 
+    /** Summary tag text. */
+    private static final String SUMMARY_TEXT = "@summary";
+
     /** Set of allowed Tokens tags in summary java doc. */
     private static final Set<Integer> ALLOWED_TYPES = Collections.unmodifiableSet(
             new HashSet<>(Arrays.asList(JavadocTokenTypes.TEXT,
-                    JavadocTokenTypes.WS))
+                    JavadocTokenTypes.WS,
+                    JavadocTokenTypes.DESCRIPTION,
+                    JavadocTokenTypes.TEXT))
+    );
+
+    /**
+     * Set of allowed Tokens tags in summary java doc.
```

Comment:
`Set of allowed inline tags in summary java doc.`

Reply:
Done.